evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -1,3 +1,4 @@
+import re
```

Comment:
something I just thought of to test: adding estimator within cg but not at end and calling `transform` :0

Reply:
This is interesting, but I was only able to see this being used on tests in which the estimator was the final component?

Reply:
Oh I just added this comment here to remind myself when I looked over this file, has nothing to do with the import 😂 

Importing `re` is just used for matching expected exceptions!